prediction_market_agent_tooling/markets/omen/omen.py

```
@@ -832,10 +840,12 @@
     language: str,
     outcomes: list[str],
     auto_deposit: bool,
-    fee: float = OMEN_DEFAULT_MARKET_FEE,
+    finalization_timeout: timedelta = timedelta(days=1),
+    fee_perc: float = OMEN_DEFAULT_MARKET_FEE_PERC,
+    distribution_hint: list[OmenOutcomeToken] | None = None,
     collateral_token_address: ChecksumAddress = WrappedxDaiContract().address,
```

Ruff

Do not perform function call `WrappedxDaiContract` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)